x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/rules_tables.tsx

```diff
@@ -63,6 +64,7 @@ const NO_ITEMS_MESSAGE = (
 export const RulesTables = React.memo<RulesTableProps>(({ selectedTab }) => {
   const [{ canUserCRUD }] = useUserData();
   const hasPermissions = hasUserCRUDPermission(canUserCRUD);
+  const isUpgradingSecurityPackages = useIsUpgradingSecurityPackages();
```

Comment:
In testing I found we must have had a regression of https://github.com/elastic/kibana/pull/103568 at some point, and now both load buttons don't disable and can be subsequently clicked to install a duplicate copy of the prebuilt rules:

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/218205244-b085d69f-0a93-43a2-adb9-e2cf73b4e088.gif" />
</p>




Can we use the new `useIsUpgradingSecurityPackages` hook to set the other `load prebuilt rules` button as disabled/loading once the first is clicked?

https://github.com/elastic/kibana/blob/5ad2a36305cda161f2d3c40cdc7a11214ab3f7fd/x-pack/plugins/security_solution/public/detections/components/rules/pre_packaged_rules/load_prepackaged_rules_button.tsx#L75-L76

Reply:
Nice catch, @spong!

Fixed that by adding a `useIsInstallingPrePackagedRules` hook that checks the prepackaged rules installation mutation (it's a different endpoint than the one we use for the package installation itself). But that was an easy fix anyway:

https://user-images.githubusercontent.com/1938181/218472168-7fe27965-0d65-47da-aeaf-a1b7ee8cd552.mov